encoder_native_test.go

```diff
@@ -226,6 +242,22 @@ func TestEncoder_Float64(t *testing.T) {
 	assert.Equal(t, []byte{0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0xF2, 0x3F}, buf.Bytes())
 }
 
+func TestEncoder_Float64FromFloat32(t *testing.T) {
+	defer ConfigTeardown()
+
+	schema := "double"
+	buf := bytes.NewBuffer([]byte{})
+	enc, err := avro.NewEncoder(schema, buf)
+	assert.NoError(t, err)
+
+	err = enc.Encode(float32(1.15))
+
+	assert.NoError(t, err)
+	expectedBytes := []byte{0x0, 0x0, 0x0, 0x60, 0x66, 0x66, 0xf2, 0x3f}
+	assert.Equal(t, expectedBytes, buf.Bytes())
+	assert.InDelta(t, 1.15, math.Float64frombits(binary.LittleEndian.Uint64(expectedBytes)), 0.0000001)
```

Comment:
What is this line for?

Reply:
Whoops, I meant for that to compare `buf.Bytes()` was approximately equal to the original input. It was mostly a sanity check, but since we're already testing the output bytes, it doesn't add anything. I can remove